docs/docs/agents/agents.md

```diff
@@ -112,6 +112,59 @@ agent.invoke(
 )
 ```
 
+## Memory
+
+To allow multi-turn conversations with an agent, you need to enable LangGraph's [persistence](../concepts/persistence.md) by providing a `checkpointer` when creating an agent:
+
+```python
+from langgraph.prebuilt import create_react_agent
+from langgraph.checkpoint.memory import InMemorySaver
+
+# highlight-next-line
+checkpointer = InMemorySaver()
+
+agent = create_react_agent(
+    model="anthropic:claude-3-7-sonnet-latest",
+    tools=[get_weather],
+    # highlight-next-line
+    checkpointer=checkpointer
+)
+
+# Run the agent
+# highlight-next-line
+config = {"configurable": {"thread_id": "1"}}
+sf_response = agent.invoke(
+    {"messages": "what is the weather in sf"},
+    # highlight-next-line
+    config
+)
+ny_response = agent.invoke(
```

Comment:
It's not obvious why this shows multi-turn interaction. There needs to be an explanation that it responds about the weather even even though it wasn't included in the query.

One could argue that it would do that by default b/c it only has access to the get weather tool (though I'm not sure)